### ISSUE TITLE
Unable to enable workflow: "Validation Failed"

### ISSUE THREAD

**Issue by wdhdev** [n8n-io/n8n#12296]
### Bug Description

When I attempt to enable my workflow which uses a GitHub trigger, it returns `Workflow could not be activated: Validation Failed` even though it is valid and was working fine before.

### To Reproduce

1. Create a new workflow using the GitHub trigger
2. Use an If statement
3. Use a step to send a message to a Discord channel
4. Workflow should fail

Here is a screenshot of my workflow:
![image](https://github.com/user-attachments/assets/09a3a0ab-d979-4e61-b04a-c245d493d705)

Note: When I run "test step" individually on each step one by one, the workflow works completely as expected, so I am unsure of the issue here as the error message is super vague and provides zero context.

I can export the workflow if needed, but I would rather not do it publicly due to some sensitive information being shown in the export.

I think this issue may be related to #10074.

### Expected behavior

The workflow should activate and run fine.

### Operating System

Debian 12.8

### n8n Version

1.71.3

### Node.js Version

20.18.0

### Database

SQLite (default)

### Execution mode

main (default)

**Comment by Joffcom** [comment]
Hey @wdhdev,

We have created an internal ticket to look into this which we will be tracking as "N8N-7990"

**Comment by Joffcom** [comment]
Hey @wdhdev,

Did you copy the node from another workflow or did you add it from the node panel?

**Comment by wdhdev** [comment]
I copied it to another workflow because the original one was not working. Should I try creating them brand new?

**Comment by movies0** [comment]
I’m experiencing an issue when trying to enable a workflow in the application. Upon clicking the enable button, an error message appears stating "Validation Failed". The workflow cannot be activated, and I am unsure what is causing the problem.

Steps to Reproduce:

Open the application and navigate to the workflows section.
Create or select an existing workflow.
Attempt to enable the workflow by clicking the enable button.
Observe the error message: "Validation Failed."
Expected Behavior:
The workflow should enable successfully without any errors, assuming all required fields and configurations are set up correctly.

Actual Behavior:
The workflow fails to enable, and the error message "Validation Failed" appears without further details.

Environment:

Application Version: (e.g., 1.5.3)
OS: (e.g., Windows 10, macOS Ventura, Ubuntu 22.04)
Browser/Platform: (if applicable, e.g., Chrome Version 118.0.0)
Workflow Details:
Workflow name: [Provide the name or description]
Workflow type: [e.g., automation, integration, etc.]
Configurations: [List any specific settings or parameters used in the workflow.]
Additional Context:

The issue persists even after restarting the application.
I verified that all required fields for the workflow are filled in.
There is no detailed explanation in the error logs or UI for why the validation failed.
Troubleshooting Steps Tried:

Rechecked all workflow configurations.
Cleared browser cache (if applicable).
Tried using a different browser/platform.
Request for Assistance:
Could you please provide guidance on how to resolve this issue or clarify what causes the "Validation Failed" error? If more information is needed, I’m happy to provide logs or additional details.

**Comment by Joffcom** [comment]
@wdhdev yeah, Can you make new ones and try again it could be that for some reason the existing webhook didn't get cleared after a restart.

**Comment by Joffcom** [comment]
Hey @wdhdev,

As we haven’t heard back from you, we’re going to close this issue to keep things organized. But don’t worry—if you have more information or need further assistance, you’re welcome to reopen the issue anytime or create a new one. We’re always happy to help! 😊

Thanks for reaching out and have a great day!
